eng/pipelines/templates/steps/build-conda-artifacts.yml

```diff
@@ -53,21 +53,28 @@ steps:
       displayName: 'Build Source Distribution for ${{ artifact.name }}'
       inputs:
         scriptPath: 'scripts/devops_tasks/build_conda_artifacts.py'
-        arguments: '-d "$(conda.output)" -b "$(conda.build)" -m "$(Build.SourcesDirectory)/sdk/${{ parameters.ServiceDirectory }}/${{ artifact.meta_source }}" -r "${{ artifact.common_root }}" -n "${{ artifact.name }}" -s "${{ parameters.ServiceDirectory }}" -o "${{ upper(parameters.ServiceDirectory) }}_SOURCE_DISTRIBUTION"'
+        arguments: >-
+          -d "$(conda.output)"
+          -b "$(conda.build)"
+          -m "$(Build.SourcesDirectory)/sdk/${{ parameters.ServiceDirectory }}/${{ artifact.meta_source }}"
+          -r "${{ artifact.common_root }}"
+          -n "${{ artifact.name }}"
+          -s "${{ parameters.ServiceDirectory }}"
+          -e "$(Build.SourcesDirectory)/eng/environment.yml"
+          -c "$(Build.SourcesDirectory)/sdk/${{ parameters.ServiceDirectory }}/ci.yml"
 
     - bash: |
         echo "##vso[task.prependpath]$CONDA/bin"
       displayName: 'Prepend PATH with Conda and INIT'
 
     - bash: |
-        conda create --yes --quiet --name ${{ artifact.name }}
-        source activate ${{ artifact.name }}
+        conda env create --name ${{ artifact.name }} --file $(Build.SourcesDirectory)/eng/environment.yml
```

Comment:
Do you expect to add other variables into environment.yml? If not should we just pass through the version variable instead of needing to parse the yml file itself? Or is the yml file for other scenarios as well?

Reply:
Eventually, yes I do expect to add other variables/configurations. Aside from that, this method is the method recommended by `conda` documentation. Normally I'd change in response to your feedback, but I'd really rather keep this guy.

Reply:
Sounds good I just wanted to make sure we weren't adding the extra complications without some reasons. The only other question I'd have is should we name the file conda_env.yml or something that indicates it should be specific to conda.

Reply:
Honestly. I was kinda leaning that way :) 

I'll do that before merging this.